src/20_Components/amp-bind.html

```diff
@@ -91,6 +97,13 @@
   </amp-img>
 
   <!--
+  The src of the video will change when the `currentAnimal` variable
```

Comment:
Maybe use [this](https://pixabay.com/en/videos/border-collies-dogs-pets-playing-7259/) (matches the border collie image above).